specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json

```diff
@@ -0,0 +1,1095 @@
+          "readOnly": true
+        },
+        "clusterArmId": {
+          "type": "string",
+          "description": "Managed Cluster ARM ID for the private link scope  (Required)"
+        },
+        "privateLinkScopeId": {
+          "readOnly": true,
+          "description": "The Guid id of the private link scope.",
+          "type": "string"
+        },
+        "privateEndpointConnections": {
+          "readOnly": true,
+          "description": "The collection of associated Private Endpoint Connections.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnectionDataModel"
+          }
+        }
+      }
+    },
+    "KubernetesConfigurationPrivateLinkScopeListResult": {
+      "type": "object",
+      "description": "Describes the list of Azure Arc PrivateLinkScope resources.",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Azure Arc PrivateLinkScope definitions.",
+          "items": {
+            "$ref": "#/definitions/KubernetesConfigurationPrivateLinkScope"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to get the next set of Azure Arc PrivateLinkScope definitions if too many PrivateLinkScopes where returned in the result set."
+        }
+      }
+    },
+    "PrivateLinkResourceListResult": {
+      "type": "object",
+      "description": "A list of private link resources",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateLinkResource": {
+      "type": "object",
+      "description": "A private link resource",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkResourceProperties",
+          "description": "Resource properties."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "PrivateLinkResourceProperties": {
+      "type": "object",
+      "description": "Properties of a private link resource.",
+      "properties": {
+        "groupId": {
+          "description": "The private link resource group id.",
```

Comment:
>The private link resource group id [](http://example.com/codeflow?start=26&length=34)

Would be good to describe the nature of this identifier (e.g. whether it's an ARM resourceId or guid, or...)? If it's an ARM resourceId, then groupResourceId would be a better name.